CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # The Changelog
 
+## Version 3.0.0: Unreleased
+- drops support for ruby > 3.1, rails < 7
```

Comment:
Is this `> 3.1` correct?
Feels like it should be `< 3.1`
If that first impression is wrong, maybe you could add more text to explain

Reply:
that's right